### PR TITLE
Fixed interference classification GUI bugs

### PR DESCRIPTION
**Pull request by madelinecoleman1**

### CODE REVIEW

File: examples/07-EMIT/interference_gui.py [hunk 1]
```diff
@@ -34,15 +34,20 @@ def install(package):
...
     if package not in installed_packages:
         install(package)
 
-# Import PySide2 and openpyxl libraries
-from PySide2 import QtWidgets, QtUiTools, QtGui
+# Import PySide6 and openpyxl libraries
+from PySide6 import QtWidgets, QtUiTools, QtGui
 from openpyxl.styles import PatternFill
 import openpyxl
+# import PySide6
```
Comment (MaxJPRey) [inline examples/07-EMIT/interference_gui.py:46]:
```suggestion
```

**Comment by myoung301** [comment]
@madelinecoleman1 I started the interference_gui.py and selected the Apache project from the Examples/EMIT folder. It appeared to load the project and then it closed AEDT and ended the script with no apparent error output. Can you give this a try and see if you see the same behavior?

**Comment by madelinecoleman1** [comment]
> @madelinecoleman1 I started the interference_gui.py and selected the Apache project from the Examples/EMIT folder. It appeared to load the project and then it closed AEDT and ended the script with no apparent error output. Can you give this a try and see if you see the same behavior?

Are lines 489 and 490 uncommented? I currently have them commented out for testing as the build machine has no UI

**Comment by myoung301** [comment]
@madelinecoleman1 You were right - I hadn't uncommented those lines. I did uncomment them and I can run the UI now. However, when I try the following steps, I get an error:

1. Open the EMIT Apache example in 2023 R2.
2. Save As to a new folder.
3. Close AEDT.
4. Verify there are no lock files.
5. Run this toolkit's UI and load the project from the save as location.
6. Click generate results.

The following error is shown in the console:
image

Can you see if you can reproduce this?

**Comment by Samuelopez-ansys** [comment]
Hi @madelinecoleman1 @myoung301 ,

Are you sure you want to include a UI example inside PyAEDT? Maybe this could be another repository, like this one:

https://github.com/ansys/pyaedt-toolkit-template

We are creating different toolkits based on Flask and PySide6.

Please reach out if you are interested.

@maxcapodi78

**Review by maxcapodi78 — APPROVED** [review]
I'll approve and try to generate documentation to see if it works.

**Pull request merged**